app/uk/gov/hmrc/agentclientrelationships/controllers/RelationshipsController.scala

```diff
@@ -236,30 +236,22 @@ class RelationshipsController @Inject()(
       }
   }
 
-  def getInactiveItsaRelationshipsAgent: Action[AnyContent] = Action.async { implicit request =>
-    withAuthorisedAsAgent { arn =>
-      findService.getInactiveItsaRelationshipForAgent(arn).map { relationships =>
-        if (relationships.nonEmpty) Ok(Json.toJson(relationships))
-        else NotFound
+  def cleanCopyStatusRecord(arn: Arn, mtdItId: MtdItId): Action[AnyContent] = Action.async { implicit request =>
+    checkOldAndCopyService
+      .cleanCopyStatusRecord(arn, mtdItId)
+      .map(_ => NoContent)
+      .recover {
+        case ex: RelationshipNotFound => NotFound(ex.getMessage)
       }
-    }
   }
 
-  def getInactiveVatRelationshipsAgent: Action[AnyContent] = Action.async { implicit request =>
+  //Generic
```

Comment:
we can remove this comment - this doesn't tell anything important :)

Reply:
ok